addon/-private/utils/json-helpers.js

```diff
@@ -23,7 +23,7 @@ export function isJsonString(str) {
  * @private
  */
 export async function parseJSON(response) {
-  const responseType = response.headers.get('content-type');
+  const responseType = response.headers.get('content-type') || 'text/xml';
```

Comment:
@scottfisk what does setting a default here get us? Would we not want the browsers to do their own thing if nothing is set?

Reply:
I just ran into this issue again and realized I never replied. Sorry about that!

So per issue #12 chrome and other modern browsers set a default of `text/xml` when the server does not provide a `content-type`. IE11, however, does not provide a default `content-type` when the server does not provide one. 

Our server does not provide a `content-type` on `DELETE` requests, it is just an empty 204.  This triggers a js error in ie11 here: https://github.com/expel-io/ember-ajax-fetch/blob/master/addon/-private/utils/json-helpers.js#L39 as we are performing an `.includes` check on an undefined object. 

Let  me know if you think there would be a better approach for an `undefined` `content-type` as to not have IE11 break.

Reply:
@scottfisk no worries! Honestly, my preference would be to not support IE at all, I don't want to open that can of worms. My concern with setting a default is if browsers change the default later, then we have the wrong default set here.

Reply:
Yeah that makes sense. 

What do you think the best way to handle a `204_NO_CONTENT` response that does not need parsing would be? Maybe here https://github.com/expel-io/ember-ajax-fetch/blob/master/addon/-private/utils/json-helpers.js#L39 changing the if statement to be `if (responseType && responseType.includes('json')) {` would be another approach to fix the issue?

Reply:
what if it's just set to an empty string? For 204 cases we'd want to go the "text" response anyway.

Reply:
I'll admit, I am ignorant to what the "correct" thing to do would be here. Most of our logic was copied from ember-ajax, so I think we probably don't want to deviate without a good reason, so migration stays close to 1:1.

Reply:
So it Looks like `ember-ajax` uses a default of `Empty Content-Type` if the header is missing. I am happy to modify the PR to do that. Reference:  https://github.com/ember-cli/ember-ajax/blob/c178c5e28a316a23cd1da5736c0e29621d838cb1/addon/mixins/ajax-request.ts#L644-L645

Reply:
@scottfisk that seems reasonable, and has the bonus of surfacing to the developer that it was empty. Let's do that 👍 

Thanks for tracking that down!